templates/home/_fsf_moos.html

```diff
@@ -2,16 +2,46 @@
   <div class='col-6'>
     <h4>Why are snaps good for MOOS projects?</h4>
     <ul>
-      <li>Snaps are easy to discover and install. Millions of users can browse and install snaps graphically in the Snap Store or from the command-line.</li>
-      <li>Snaps install and run the same across Linux. They bundle the exact version of MOOS/MOOS-IvP required and your app’s library dependencies.</li>
-      <li>Snaps automatically update to the latest version. Four times a day, users’ systems will check for new versions and upgrade in the background.</li>
-      <li>Upgrades are not disruptive. Because upgrades are not in-place, users can keep your app open as it’s upgraded in the background.</li>
-      <li>Upgrades are safe. If your app fails to upgrade, users automatically roll back to the previous revision.</li>
+      <li>Bundle all the runtime requirements, including the exact version of MOOS/MOOS-IvP and system libraries you need.</li>
+      <li>Expand the distributions supported beyond just Ubuntu.</li>
+      <li>Directly and reliably control the delivery of application updates using existing infrastructure.</li>
+      <li>Extremely simple creation of daemons.</li>
     </ul>
   </div>
 
+  <div class='col-6'>
+    <h4>How moos-ping defines snapcraft.yaml:</h4>
```

Comment:
This fits the pattern established by the other languages/frameworks, but I'm not sure it makes sense here given that "moos-ping" is not an actual thing. It's just an example; no one's going to know what "moos-ping" is. Would "Here's an example MOOS snapcraft.yaml" (or something similar) work better?

Reply:
If I understand correctly, MOOS is a communication mechanism through which apps interface. So this might be semantically "incorrect" but it would make more sense to newcomers to the guide. I will make the change.

Reply:
This should be updated to match the style used in the ROS first snap flow:
>Here's an example snapcraft.yaml that uses MOOS: